app/javascript/packages/verify-flow/verify-flow-troubleshooting-options.tsx

```diff
@@ -0,0 +1,40 @@
+import { useContext } from 'react';
+import { TroubleshootingOptions } from '@18f/identity-components';
+import { getConfigValue } from '@18f/identity-config';
+import { t } from '@18f/identity-i18n';
+import FlowContext from './context/flow-context';
+
+function VerifyFlowTroubleshootingOptions() {
+  const { inPersonURL } = useContext(FlowContext);
+
+  return (
+    <>
+      <TroubleshootingOptions
+        heading={t('components.troubleshooting_options.default_heading')}
+        options={[
+          {
+            url: 'https://login.gov/contact/',
+            text: t('idv.troubleshooting.options.contact_support', {
+              app_name: getConfigValue('appName'),
+            }),
+            isExternal: true,
+          },
+        ]}
+      />
+      {inPersonURL && (
```

Comment:
cc @tomas-nava @sheldon-b looks like we're adding one more page that links to IPP, something to consider when adding the intro + location select in React